### ISSUE TITLE
[D1+B1] Asset icon should be rendered above asset unit

### ISSUE THREAD

**Issue by ipaleka** [asastats/channel#982]
By mochanerd in [Discord](https://discord.com/channels/906917846754418770/915342738201931836/1222984607767728190):

> UI update request: please center the logos above the ticker symbols, the overlap looks unaesthetic on both Android internet browser and the App itself

<img src="https://github.com/asastats/channel/assets/49662536/cd26287f-ef64-403b-8ad8-9836c1ee4ebe" width=400>

**Comment by kerrilija** [comment]
This must also be done on the website. But we will have to enlarge the list tile in height or make asset icon tiny. Both approaches are not the best, I'm not sure we should be doing this at all.. At least before a UI revamp.

**Comment by ipaleka** [comment]
I'm unsure why you think we'd need to enlarge height/decrease icon size? 

As I see this request for the [website](https://www.w3schools.com/cssref/pr_pos_z-index.php), the ASASTATS icon should be rendered above the ASASTATS text in the image above.

**Comment by kerrilija** [comment]
Yes, it should be rendered but there is no space, both in website and mobile app. Do you see other options, other than those we mentioned?

<img src="https://github.com/user-attachments/assets/a283ef4d-d53d-422d-b524-1f745d27e3c0" width=400>

**Comment by ipaleka** [comment]
For me personally, what is presented in that image is absolutely the best solution.

**Comment by kerrilija** [comment]
I misunterstood the issue completely, now I get it => zIndex :)

**Comment by ipaleka** [comment]
I'm not sure if this has been resolved or if it hasn't been the issue on the website:

<img src="https://github.com/user-attachments/assets/c0f50578-b247-450a-89b4-e23b7324a4f2" width=300>
<img src="https://github.com/user-attachments/assets/e265b1da-2964-40e1-89c2-fafc4538e754" width=300>

**Comment by kerrilija** [comment]
It is always the same (as it should be) on the website. Just a mobile-app thing

**Comment by ipaleka** [comment]
> It is always the same (as it should be) on the website. Just a mobile-app thing

I misunderstood [this](https://github.com/asastats/channel/issues/982#issuecomment-2616845590):

> This must also be done on the website.

**Comment by kerrilija** [comment]
Fixed and awaiting deployment.

[D1] kerrilija
[B1] mochanerd
